### ISSUE TITLE
playback suddenly stops

### ISSUE THREAD

**Issue by cukabeka** [PaulWoitaschek/Voice#651]
Hi

Since the update to Voice, my beloved Material Player started behaving strangely. It stops the playback randomly. I can press play again, and then it resumes but stops mostly again after some seconds. I can't reproduce the behaviour, though.

On Material Player, the same files player flawlessly.

App-Version: 3.5.0.1
Phone Name: Huawei P9
Android Version: 7.0

BTW: I really dont dig the new Logo. ;)

**Comment by miguelfc** [comment]
+1 It happens to me too.

In my case:
App-Version: 3.5.0.1
Phone Name: Xperia Z1 Compact
Android Version: 5.1

**Comment by illarionds** [comment]
Me too.

App: 3.5.0.1
Samsung Galaxy S3
Android 4.3

**Comment by cukabeka** [comment]
I noticed that if I cancel/kill all other apps before starting Voice, the error is less likely to happen. Maybe it's about RAM?

**Comment by Steffen3000** [comment]
Voice Audiobook Player is a great app for playing my downloaded podcasts.

It works fine on Samsung Galaxy S7 - Android 8, connected to Android Auto, but on my older phone that I often use for podcasting it close down and restart after app.2 minuts, running Android 5.0 on HTC One connected to Android Auto.

**Comment by PaulWoitaschek** [comment]
Thanks for your report. 
I assume this issue is fixed as this issue is ancient and a lot changed in the meantime. If not please open a new issue.
